ee/test/test_migration_0010.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from posthog.test.base import TestMigrations
+
+
+@pytest.mark.ee
+class TagsTestCase(TestMigrations):
+
+    migrate_from = "0009_null_definition_descriptions"  # type: ignore
+    migrate_to = "0010_migrate_definitions_tags"  # type: ignore
+
+    @property
+    def app(self):
+        return "ee"
+
+    def setUpBeforeMigration(self, apps):
+        EnterpriseEventDefinition = apps.get_model("ee", "EnterpriseEventDefinition")
+        EnterprisePropertyDefinition = apps.get_model("ee", "EnterprisePropertyDefinition")
+
+        self.event_definition = EnterpriseEventDefinition.objects.create(
+            team_id=self.team.id, name="enterprise event", deprecated_tags=["a", "b", "c", "a", "b"]
+        )
+        self.property_definition_with_tags = EnterprisePropertyDefinition.objects.create(
+            team_id=self.team.id, name="property def with tags", deprecated_tags=["b", "c", "d", "e", "e"]
+        )
+        self.property_definition_without_tags = EnterprisePropertyDefinition.objects.create(
+            team_id=self.team.id, name="property def without tags",
+        )
+
+    def test_tags_migrated(self):
+        Tag = self.apps.get_model("posthog", "Tag")  # type: ignore
```

Comment:
Let's add some existing tags here, so that we can test the case where the other migration runs first.